mosfit/models/magnetar/magnetar.json

```diff
@@ -26,33 +26,6 @@
             }
         }
     },
-    "ebv":{
```

Comment:
So these lines are in the JSON file to facilitate running the code in a "parameter scan" mode where they might be randomly varied. Any reason these were deleted in this PR, were they breaking something?